kitsune/wiki/api.py

```diff
@@ -19,10 +19,13 @@ class Meta:
 
 
 class DocumentDetailSerializer(DocumentShortSerializer):
+    summary = serializers.CharField(source='summary', read_only=True)
+    url = serializers.CharField(source='get_absolute_url', read_only=True)
+
     class Meta:
         model = Document
-        fields = ('id', 'title', 'slug', 'locale', 'products', 'topics',
-                  'html')
+        fields = ('id', 'title', 'slug', 'url', 'locale', 'products', 'topics',
+                  'summary', 'html')
```

Comment:
I had to add the summary and url fields to this serializer so that it had all the bits the previous results had.

Does this affect other APIs adversely?